nvbench/option_parser.cu

```diff
@@ -773,17 +749,16 @@ catch (std::exception &e)
                 e.what());
 }
 
-void option_parser::disable_blocking_kernel()
+void option_parser::enable_profile()
 {
-  // If no active benchmark, save args as global.
-  if (m_benchmarks.empty())
+  // If no active benchmakr, save args as global
   {
```

Comment:
critical: as written, we never call `set_run_once(true)` so option has no effect. I think it should be:
```
if (m_benchmarks.empty())
{
 ...
```

Reply:
Verifying that using `--profile` options results in single run of the benchmark:

```
(nvbench) opavlyk@ee09c48-lcedt:~/repos/nvbench$ ./build/bin/nvbench.example.cpp20.axes -b simple -d 0 --profile | grep "Pass: Cold"
Pass: Cold: 1.006560ms GPU, 1.009277ms CPU, 0.00s total GPU, 0.00s total wall, 1x 
```

```
(nvbench) opavlyk@ee09c48-lcedt:~/repos/nvbench$ ./build/bin/nvbench.example.cpp20.axes -b simple -d 0 | grep "Pass: Cold"
Pass: Cold: 1.002844ms GPU, 1.011917ms CPU, 0.50s total GPU, 0.52s total wall, 499x 
```